manim/mobject/number_line.py

```diff
@@ -68,6 +68,29 @@ class NumberLine(Line):
     kwargs : Any
         Additional arguments to be passed to :class:`~.Line`.
 
+    Examples
+    --------
+    .. manim:: NumLineExample
+        :save_last_frame:
+        class NumLineExample(Scene):
+            def construct(self):
+                l0 = NumberLine(x_range=[-10, 10, 2], length=10, color=BLUE, include_numbers=True, label_direction=UP)
+                l0.shift(2 * UP)
```

Comment:
I think it'd make more sense if each of the lines were added to a VGroup and then arranged via `.arrange()`.

Here's what it could look like:
```py
line_group = VGroup(l0, l1, l2, l3).arrange(DOWN, buff = 1)
self.add(line_group)
```
This would automate the effort of settting the position via the shifts (and make the code look neater).